src/miniinspector.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import MiniInspectorUI from './miniinspectorui';
+
+export default class MiniCKEditorInspector {
```

Comment:
We're missing docs here.

Reply:
Done. The only thing to consider here is to maybe not mention multiple editors option since it's really made with only one editor in mind (since no editors dropdown nor any inspector UI).